src/pages/questions/Result.tsx

```diff
@@ -1,26 +1,110 @@
 import React, { FC } from 'react';
-import { Typography } from '@mui/material';
-import { getKoefTarget, getAf } from './utils';
 import { FormAnswers } from '../../config';
+import { getResult } from './utils';
+import SleepCard from './SleepCard';
+import KclCard from './KclCard';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemText from '@mui/material/ListItemText';
+import Avatar from '@mui/material/Avatar';
+import PlanCard from './PlanCard';
+import { Grid, Typography } from '@mui/material';
 
 type ResultProps = {
   values: FormAnswers;
 };
 
-const Result: FC<ResultProps> = ({
-  values: { weight, height, age, activity, target },
-}) => {
-  const af = getAf(activity);
-  const koefTarget = getKoefTarget(target);
-
-  const bmr = 655.0955 + 9.5634 * weight + 1.8496 * height - 4.6756 * age;
-  const tdee = bmr * af;
-  const kcl = tdee * koefTarget;
-
+const Result: FC<ResultProps> = ({ values }) => {
+  const { trainingPlanDays, kcl, sleepInfo } = getResult(values);
   return (
-    <Typography variant="h4" m={3}>
-      Tvůj denní příjem kalorií- {kcl} Kcal
-    </Typography>
+    <>
+      <List sx={{ mb: '2' }}>
+        {trainingPlanDays ? (
+          <Grid container spacing={2}>
+            <ListItem>
+              <Grid item={true} xs={3}>
```

Comment:
Replace `item={true}` with `item` (all occurences in this file) - it is preferered way to write _boolean_ props if they are _truthy_  and linter usually autofixes it